hack/make/sign-repos

```diff
@@ -26,7 +26,7 @@ sign_packages(){
 
 		# sign the repo metadata
 		for F in $(find $APTDIR -name Release); do
-			if test "$F" -nt "$F.gpg" ; then
+			if ! test -e "$F.gpg" || test "$F" -nt "$F.gpg" ; then
```

Comment:
This one also falls under "all kinds of trouble" mentioned earlier. Also apparently zsh and bash behave differently in this regard, which is very frustrating to learn:

```bash
$ echo $SHELL
/bin/zsh
$ ls
$ touch a
$ test a -nt b && echo yes || echo no
no
$ test b -nt a && echo yes || echo no
no
```

You are right though, from https://www.mkssoftware.com/docs/man1/test.1.asp:
> Note:
All existing files are considered to be newer than a nonexistent file. In turn, no nonexistent file is newer than any existing file. Two nonexistent files are considered the same age.

I'll revert this change (and the yum one below) and run some tests to verify that I'm not crazy :)